### ISSUE TITLE
Can't store to an array element

### ISSUE THREAD

**Issue by stevenrbrandt** [STEllAR-GROUP/phylanx#1236]
The following code
```
import numpy as np

@Phylanx
def foo():
    z = np.zeros((3,4))
    z[0][0] = 1

foo()
```
fails with the error
```
  File "k.py", line 9, in <module>
    foo()
  File "/home/jovyan/.local/lib/python3.6/site-packages/phylanx-0.0.1-py3.6-linux-x86_64.egg/phylanx/ast/transducer.py", line 168, in __call__
    result = self.backend.call(*mapped_args, **mapped_kwargs)
  File "/home/jovyan/.local/lib/python3.6/site-packages/phylanx-0.0.1-py3.6-linux-x86_64.egg/phylanx/ast/physl.py", line 578, in call
    self.wrapped_function.__name__, *args, **kwargs)
RuntimeError: k.py(7, 4): slice:: store function should only be called for the primitives that support it (e.g. variables): HPX(invalid_status)
```

**Comment by hkaiser** [comment]
@stevenrbrandt The Phylanx frontend understands (multi-) indexing an array using `z[0, 0] = 1` only, at this time. I'll see what I can do to support `z[0][0] = 1` as well.

**Comment by stevenrbrandt** [comment]
@hkaiser I had tried it both ways from Python. Maybe what's needed is a change to the Python parser.

**Comment by hkaiser** [comment]
> @hkaiser I had tried it both ways from Python. Maybe what's needed is a change to the Python parser.

@stevenrbrandt please see #1240 for a possible fix

**Comment by hkaiser** [comment]
This is not fixed on master (#1240 is not merged yet)
